.github/workflows/update-syscalls.yml

```diff
@@ -0,0 +1,39 @@
+name: Update Syscalls
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '0 2 1 * *'
+
+jobs:
+  update-syscalls:
+    runs-on: ubuntu-latest
+    container:
+      image: falcosecurity/syscalls-bumper:latest
+    permissions:
+      contents: write
+      pull-requests: write
+    steps:
+      - name: Git set-up
+        run: |
+          apk update && apk add git
+
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
```

Comment:
I think we might want to clone without fetch-depth 0 here, as it is useless!
I'd instead add the `ref: master` so that even when manually dispatched  from the wrong base, it should still work fine.